python/pycylon/pycylon/data/aggregates.pxd

```diff
@@ -30,12 +33,57 @@ cdef extern from "../../../../cpp/src/cylon/compute/aggregate_kernels.hpp" names
         CQUANTILE 'cylon::compute::QUANTILE'
         CSTDDEV 'cylon::compute::STDDEV'
 
+    cdef cppclass CBasicOptions "cylon::compute::BasicOptions":
+        CBasicOptions()
+        CBasicOptions(cbool skip_nulls)
+
+    cdef cppclass CVarKernelOptions "cylon::compute::VarKernelOptions":
+        CVarKernelOptions()
+        CVarKernelOptions(int ddof, cbool skip_nulls)
+
+
 cdef extern from "../../../../cpp/src/cylon/compute/aggregates.hpp" namespace "cylon::compute":
+    CStatus Sum(const shared_ptr[CTable] & table, int col_idx, shared_ptr[CTable] & output)
```

Comment:
nit: there is spacing between the `&` and the var. As I remember the code formatter introduces this. We should probably get rid of that space. WDYT?

Reply:
This is repeated in similar codeblocks.